js/app.js

```diff
@@ -1,239 +1,174 @@
 'use strict';
 
-// for (var p = 0; p < productNames.length; p++)
-var productNames = ['babySweep', 'bag', 'banana', 'bathroom', 'boots', 'breakfast', 'bubblegum', 'chair','cthulhu', 'dogDuck', 'dragon', 'pen', 'petSweep', 'scissors', 'shark', 'tauntaun', 'unicorn', 'usb', 'waterCan', 'wineGlass' ]
-// }
-console.log(productNames);
-
-var productName = [];
-var productImage1 = '';
-var productImage2 = '';
-var productImage3 = '';
-
-//TODO automate var from Array
-// var arrayOfObjects = [];
-// for (var i=0; i < )
+//Inspired often by Sugey & Jon & the first 24
+///var///////////
+var product;
+var img;
+var newImg;
+var pushImg;
+var img1;
+var img2;
+var img3;
 var attempts = 0;
 var maxAttempts = 25;
-var babySweep = new ProductName('babySweep');
-var bag = 0;
-var banana = 0;
-var bathroom = 0;
-var boots = 0;
-var breakfast = 0;
-var bubblegum = 0;
-var chair = 0;
-var cthulhu = 0;
-var dogDuck = 0;
-var dragon = 0;
-var pen = 0;
-var petSweep = 0;
-var scissors = 0;
-var shark = 0;
-var tauntaun =0;
-var unicorn = 0;
-var usb = 0;
-var waterCan = 0;
-var wineGlass = 0;
+//get placing from website///
+var productImageParent = document.getElementById('ImageAll');
+var productImg1 = document.getElementById('img1');
+var productImg2 = document.getElementById('img2');
+var productImg3 = document.getElementById('img3');
 
-var productImageParent = document.getElementById('productImage');
-var responseParent = document.getElementById('response');
 
-function setup() {
-  //think about what does setup do
-  //name var = element(object)
-  productImage1 = generateRandomProduct();
-  productImage2 = generateRandomProduct();
-  productImage3 = generateRandomProduct();
-//TODO make random not equal last 3 nor other 2
-  if (attempts) {
-    productImageParent.removechild(productImageParent.lastChild);
-    productImageParent.removechild(productImageParent.lastChild);
-    productImageParent.removechild(productImageParent.lastChild);
-  }
-  // for (var p = 0; p < 3; i++) {
-  //   productNames ;
-  // }
-  renderProductImage(productImage1);
-  renderProductImage(productImage2);
-  renderProductImage(productImage3);
-  // updateAnswer();
-  // updateAttempts();
+///calls////
+renderProductImage();
```

Comment:
I'm not sure why you need so many calls right here.  Also, I'm not clear on what a lot of these functions actually do.